consul/install.sls

```diff
@@ -0,0 +1,49 @@
+{% from "consul/map.jinja" import consul, consul_service with context %}
+
+# At the time of writing, Hashicorp doesn't maintain any official package
+# repositories for common linux distributions. The recommended installation
+# method is to download a zip, verify it, and extract consul into the system
+# path.
+
+include:
+  - .service
+  - .config
+
+install_consul_binary:
+  archive.extracted:
+    - name: /usr/bin/
+    - source: {{ consul.consul_zip }}
+    - source_hash: {{ consul.consul_zip_checksum }}
+    - archive_format: zip
+    - if_missing: /usr/bin/consul
+
+permission_consul_bin:
+  file.managed:
+    - name: /usr/bin/consul
+    - mode: 0755
+    - require:
+      - archive: download_package
```

Comment:
Just need to update the id for the archive step and then it should be good to go